tests/languages/idris/keyword_feature.test

```diff
@@ -0,0 +1,17 @@
+case data do else if implementation

```

Comment:
This doesn't look like it covers all keywords. Is there a reason for this?

Reply:
Just because Idris has relatively larger size set of keywords. As lexing keywords is not so complex task, testing some of them is enough.